# The joint path point contains a vector of joint positions.
float64[] positions
